documentation/openapi.yaml

```diff
@@ -1,20 +1,21 @@
 openapi: 3.0.0
 info:
-  version: 1.0.0-oas3
+  version: 1.1.0
```

Comment:
why don't we use v1, v2 etc.?
e.g. apiurl.com/v1/

Reply:
https://semver.org/

Reply:
ok let's do it like a pro.